exploration/semantic-skeletons.md

```diff
@@ -0,0 +1,209 @@
+# Semantic Skeletons Design
+
+Status: **Proposed**
+
+<details>
+	<summary>Metadata</summary>
+	<dl>
+		<dt>Contributors</dt>
+		<dd>@sffc</dd>
+    <dd>@aphillips</dd>
+		<dt>First proposed</dt>
+		<dd>2024-04-06</dd>
+		<dt>Pull Requests</dt>
+		<dd><a href="https://github.com/unicode-org/message-format-wg/pull/1067">#1067</a></dd>
+	</dl>
+</details>
+
+## Objective
+
+_What is this proposal trying to achieve?_
+
+### Provide support for formatting date/time values using semantic skeletons
+
+"Semantic skeletons" are a method introduced in CLDR 46 for programmatically selecting a datetime pattern for formatting. 
+There is a fixed set of acceptable semantic skeletons.
+
+Previously, ICU MessageFormat provided support for "classical skeletons",
+using a microsyntax derived from familiar picture strings (see below)
+combined with code in ICU (`DateTimePatternGenerator`) to produce the desired date/time value format.
+`Intl.DateTimeFormat` uses "option bags" to provide a similar capability.
+A classical skeleton allowed users to express the desired fields and field widths in a formatted date/time value.
+The runtime uses locale data to determine minutiae such as 
+field-order, 
+separators, 
+spacing, 
+field-length, 
+etc. to produce the desired output.
+
+Advantages of semantic skeletons over classical skeletons:
+
+- Provides all and only those combinations that make sense
+   - Allows for more efficient implementation, since there is no need to support "crazy" combinations like "month-hour"
```

Comment:
```suggestion
   - Allows for more efficient implementation, since there is no need to support unlikely combinations like "month-hour"
```

Avoid quoted adjectives in favor of a more precise description, if possible. Is it really true that _no one_ would want to use these combinations, or (my guess) is it more likely to be an error than not?

Reply:
done